src/app/credential-issuer/middleware.js

```diff
@@ -2,13 +2,47 @@ const axios = require("axios");
 const {
   API_BASE_URL,
   API_REQUEST_EVIDENCE_PATH,
+  API_SHARED_ATTRIBUTES_JWT_PATH,
   EXTERNAL_WEBSITE_HOST,
 } = require("../../lib/config");
 
 module.exports = {
+  getSharedAttributesJwt: async (req, res, next) =>  {
+    try {
+      const apiResponse = await axios.get(
+        `${API_BASE_URL}${API_SHARED_ATTRIBUTES_JWT_PATH}`,
+        {
+          headers: {
+            "ipv-session-id": req.session.ipvSessionId
+          }
+        }
+      );
+
+      const jwt = apiResponse?.data?.jwt;    //will need to be confirmed
+      const base64regex = /^([0-9a-zA-Z+/]{4})*(([0-9a-zA-Z+/]{2}==)|([0-9a-zA-Z+/]{3}=))?$/;
+
+      if (!jwt) {
+        res.status(500);
+        return res.send("Missing JWT");
+      } else if (jwt.length > 6000) { // this figure can be changed
+        res.status(500);
+        return res.send("JWT exceeds maximum limit");
+      } else if (!base64regex.test(jwt)) { 
+        res.status(500);
+        return res.send("Invalid base64 encoded JWT");
+      }
+
+      req.session.jwt = jwt;
```

Comment:
Small change here, but could we change the req.session.jwt to be called something like req.session.sharedAttributesJwt - Just might make it slightly clearer when we have other JWT's going though the system.